src/Controls/src/Core/Handlers/Items2/CarouselViewHandler2.iOS.cs

```diff
@@ -83,7 +82,10 @@ protected override UICollectionViewLayout SelectLayout()
 
 				// Create our section layout
 				var section = NSCollectionLayoutSection.Create(group: group);
-				section.InterGroupSpacing = itemSpacing;
+				if (VirtualView.ItemsLayout is LinearItemsLayout linearItemsLayout)
```

Comment:
Could include a related test?

Reply:
@jsuarezruiz The test case is already present in Issue25192.cs